Incorrect return type in header files causing Doxygen to fail

### Affected Branch

trunk

### Basic Diagnostics

- [X] I've pulled the latest changes on the affected branch and the issue is still present.

- [ ] The issue is reproducible in docker

### Description

A few header files have extraneous /return commands causing Doxygen to fail.  These appear to be innocent typos due to copy/paste.

In order to reproduce the issue, follow these steps:

1.   Clone repo
```
git clone --recurse-submodules https://github.com/mit-dci/opencbdc-tx
```
2.   Run Doxygen
```
doxygen Doxyfile
```
The errors you'll eventually see are:
```
...
Generating call graph for function cbdc::transaction::wallet::seed
/Users/duc/Dev2/opencbdc-tx_dmt/src/uhs/transaction/wallet.hpp:142: error: documented empty return type of cbdc::transaction::wallet::seed_readonly (warning treated as error, aborting now)
...
Generating call graph for function cbdc::watchtower::error_cache::check_tx_id
/Users/duc/Dev2/opencbdc-tx_dmt/src/uhs/atomizer/watchtower/error_cache.hpp:36: error: documented empty return type of cbdc::watchtower::error_cache::push_errors (warning treated as error, aborting now)
Exiting...


### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/src/uhs/atomizer/watchtower/error_cache.hpp
+++ b/src/uhs/atomizer/watchtower/error_cache.hpp
@@ -34,7 +34,6 @@ namespace cbdc::watchtower {
         /// Moves an error into the error cache, evicting the oldest error if
         /// the cache has reached its maximum size.
         /// \param errs the error to move.
-        /// \return true if error was successfully added to cache. False if error is invalid and not added to cache.
         void push_errors(std::vector<tx_error>&& errs);
 
         /// Checks the cache for an error associated with the given Tx ID.

--- a/src/uhs/transaction/wallet.hpp
+++ b/src/uhs/transaction/wallet.hpp
@@ -150,7 +150,6 @@ namespace cbdc::transaction {
         /// \param value the value to use for the seeded outputs.
         /// \param begin_seed the start index to use for the seeded outputs.
         /// \param end_seed the end index to use for the seeded outputs.
-        /// \return true if setting the seed parameters succeeded.
         void seed_readonly(const hash_t& witness_commitment,
                            uint32_t value,
                            size_t begin_seed,

